ArduPlane/Log.cpp

```diff
@@ -505,7 +509,7 @@ void Plane::Log_Read(uint16_t list_entry, int16_t start_page, int16_t end_page)
 
     cliSerial->printf("%s\n", HAL_BOARD_NAME);
 
-	DataFlash.LogReadProcess(list_entry, start_page, end_page,
+   DataFlash.LogReadProcess(list_entry, start_page, end_page,
```

Comment:
Sorry, I don't know what you mean by commit squashing and tidying (beyond addressing reviewers' comments, including yours); please clarify. As for the whitespace changes, in the initial commit I fixed spacing issues in every file I touched, but then people said I shouldn't lump together space changes with "functional" changes, so I reverted the space changes in a later commit. What looks like space changes are actually the _reverse_ of the space changes I made in the initial commit. In other words, compared to the main ArduPilot branch, there shouldn't be any space changes to the existing code.